backup.sh

```diff
@@ -15,23 +16,24 @@ copy_s3 () {
   echo "Uploading ${DEST_FILE} on S3..."
   if [[ -z "$S3_PREFIX" ]]; then
     # backup without prefix
-    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$DEST_FILE && \
-    aws $AWS_ARGS s3 cp s3://$S3_BUCKET/$DEST_FILE s3://$S3_BUCKET/$DEST_LATEST_FILE
+    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$FREQUENCY/$DEST_FILE && \
+    aws $AWS_ARGS s3 cp s3://$S3_BUCKET/$FREQUENCY/$DEST_FILE s3://$S3_BUCKET/$FREQUENCY/$DEST_LATEST_FILE
```

Comment:
@vijayabharathib can we create latest alias for hourly backup outside of frequency folder. So that, that is referring always latest.

Reply:
This is done.

Reply:
@vijayabharathib I don't see any new commits for this fix. 

Currently, the latest.dump.tar.gz can be overridden my daily/weekly/monthly. 

The ask is creating alias only for hourly. 

So that latest referring latest backup.

Reply:
Got it. this is there.